packages/circularprogress/src/css/index.js

```diff
@@ -38,20 +37,20 @@ export default {
     transition: 'stroke-dashoffset 1s'
   },
   [`.psds-circularprogress__fg.psds-theme--${themeNames.light}`]: {
-    stroke: core.colors.gray04
+    stroke: colorsTextIcon.highOnLight
   },
   [`.psds-circularprogress__fg.psds-theme--${themeDefaultName}`]: {
-    stroke: core.colors.white
+    stroke: colorsTextIcon.highOnDark
   },
   // __bg
   [`.psds-circularprogress__bg`]: {
     fill: 'transparent',
     strokeWidth: `${vars.style.strokeWidth}px`
   },
   [`.psds-circularprogress__bg.psds-theme--${themeNames.light}`]: {
-    stroke: transparentize(0.8, core.colors.gray04)
+    stroke: 'rgba(0, 0, 0, 0.15)'
```

Comment:
This is the third time I've seen this sorta usage of this color for this. Maybe it should be a core token like track/progress on lo & high

Reply:
Perhaps.  We have used it a few times.  In my mind, it doesn't rise to the generality level of text, icon, and border though.

Reply:
Should we maybe create a tokens package?

Reply:
Isn't that what core is?